src/js/components/award/shared/InfoTooltipContent.jsx

```diff
@@ -1217,6 +1217,49 @@ export const CFDASectionInfo = (
     </div>
 );
 
+export const FilterTooltip = (
+    <div className="award-summary-tooltip">
+        <div className="tooltip__title" style={{ textAlign: "left" }}>
```

Comment:
this inline style can be removed without affecting the appearance of the tooltip

Reply:
done